walk-in-interview/src/main/webapp/strings.en.js

```diff
@@ -90,7 +90,10 @@ AppStrings['create-account'] = {
  * All the strings to be used in the account/create-account/applicant/index.html.
  */
 AppStrings['create-applicant-account'] = {
+  'name': 'Name',
+  'skills-title': 'Skills',
   'new-user-info': 'Account is successfully created!',
+  'non-new-user-info': 'This is a currently existing user. Log in successfully!',
```

Comment:
nit: "This is a currently existing user." <- it's a bit odd.  I don't think you've ever seen this in any website you logged into :) can you rephrase?